src/pages/quizzes.tsx

```diff
@@ -0,0 +1,142 @@
+import { useMemo, useState } from "react"
+import { GetStaticProps, InferGetStaticPropsType, NextPage } from "next"
+import { SSRConfig, useTranslation } from "next-i18next"
+import { serverSideTranslations } from "next-i18next/serverSideTranslations"
+import { FaGithub } from "react-icons/fa"
+import { Box, Flex, Icon, Stack, Text, useDisclosure } from "@chakra-ui/react"
+
+import { QuizStatus } from "@/lib/types"
+
+import { ButtonLink } from "@/components/Buttons"
+import FeedbackCard from "@/components/FeedbackCard"
+import PageMetadata from "@/components/PageMetadata"
+import QuizWidget from "@/components/Quiz/QuizWidget"
+import QuizzesList from "@/components/Quiz/QuizzesList"
+import QuizzesModal from "@/components/Quiz/QuizzesModal"
+import QuizzesStats from "@/components/Quiz/QuizzesStats"
+import { useLocalQuizData } from "@/components/Quiz/useLocalQuizData"
+import Translation from "@/components/Translation"
+
+import { trackCustomEvent } from "@/lib/utils/matomo"
+import { getRequiredNamespacesForPage } from "@/lib/utils/translations"
+
+import { ethereumBasicsQuizzes, usingEthereumQuizzes } from "@/data/quizzes"
+
+import { INITIAL_QUIZ } from "@/lib/constants"
+
+const handleGHAdd = () =>
+  trackCustomEvent({
+    eventCategory: "quiz_hub_events",
+    eventAction: "Secondary button clicks",
+    eventName: "GH_add",
+  })
+
+const QuizzesHubPage: NextPage<
+  InferGetStaticPropsType<typeof getStaticProps>
+> = () => {
+  const { t } = useTranslation()
+
+  const [userStats, updateUserStats] = useLocalQuizData()
+  const [quizStatus, setQuizStatus] = useState<QuizStatus>("neutral")
+  const [currentQuiz, setCurrentQuiz] = useState(INITIAL_QUIZ)
+  const { onOpen, isOpen, onClose } = useDisclosure()
+
+  const commonQuizListProps = useMemo(
+    () => ({
+      userStats,
+      quizHandler: setCurrentQuiz,
+      modalHandler: onOpen,
+    }),
+    [onOpen, userStats]
+  )
+
+  return (
+    <Box>
+      <PageMetadata
+        title={t("quizzes-title")}
+        description={t("quizzes-subtitle")}
+      />
+      {/* TODO: Add HubHero */}
```

Comment:
`HubHero` currently absent here as it has yet to be updated for NextJS.

Reply:
FYI it is now available 👍🏼